platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/crypto/CryptoArgsProvider.java

```diff
@@ -81,13 +87,28 @@ public static AddressBook createAddressBook(final int size) {
     @NonNull
     public static AddressBookAndCerts loadAddressBookWithKeys(final int size)
             throws URISyntaxException, UnrecoverableKeyException, KeyLoadingException, KeyStoreException,
-                    NoSuchAlgorithmException {
+                    NoSuchAlgorithmException, KeyGeneratingException, NoSuchProviderException {
         final AddressBook loadedAB = createAddressBook(size);
         final Map<NodeId, KeysAndCerts> loadedC =
                 CryptoStatic.loadKeysAndCerts(loadedAB, ResourceLoader.getFile("preGeneratedKeysAndCerts/"), PASSWORD);
         return new AddressBookAndCerts(loadedAB, loadedC);
     }
 
+    /**
+     * returns a record with the addressBook and signing keys loaded from file with generated agreement keys.
+     *
+     * @param size the size of the required address book
+     */
+    @NonNull
+    public static AddressBookAndCerts createAddressBookLoadSigKeysCreateAgrKeys(final int size)
+            throws URISyntaxException, UnrecoverableKeyException, KeyLoadingException, KeyStoreException,
+                    NoSuchAlgorithmException, KeyGeneratingException, NoSuchProviderException {
+        final AddressBook loadedAB = createAddressBook(size);
```

Comment:
this variable is named `loadedAB`, but I'm not seeing what is "loaded" about it. It seems to be simply created

Reply:
I've modified the variable name to `createdAB` and updated the variable name in the method I copied the code from that is also creating the address book instead of loading it.